supervisor/supervisor.go

```diff
@@ -333,8 +333,10 @@ func (s *Supervisor) restore() error {
 			continue
 		}
 		processes, err := container.Processes()
```

Comment:
For now `err` is always nil, can we ignore it here?
`processes, _ := container.Processes()`